ratis-proto/src/main/proto/Raft.proto

```diff
@@ -110,6 +110,7 @@ message RaftRpcRequestProto {
   bytes replyId = 2;
   RaftGroupIdProto raftGroupId = 3;
   uint64 callId = 4;
+  RoutingTableProto routingTable = 5;
```

Comment:
Let's change the field number to 14 (next to slidingWindowEntry) since it is a non-core field.